### ISSUE TITLE
ERROR - No error handlers are registered, logging exception

### ISSUE THREAD

**Issue by bigfootdeveloper** [trizin/Telegram-Airdrop-Bot#11]
I deployed all following the instructions.  The bot runs normal until it shows "telegram channel" and I press the button "Done". 

Brings this error:

```
2021-10-19 22:01:24,288 - apscheduler.scheduler - INFO - Scheduler started
2021-10-19 22:03:37,302 - telegram.ext.dispatcher - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/telegram/ext/dispatcher.py", line 432, in process_update
    handler.handle_update(update, self, check, context)
  File "/usr/local/lib/python3.7/site-packages/telegram/ext/conversationhandler.py", line 517, in handle_update
    new_state = handler.handle_update(update, dispatcher, check_result, context)
  File "/usr/local/lib/python3.7/site-packages/telegram/ext/handler.py", line 156, in handle_update
    return self.callback(update, context)
  File "bot.py", line 187, in checkCaptcha
    if CAPTCHA_DATA[user.id] != text:
KeyError: 1050587771
bot.py:219: DeprecationWarning: count is deprecated. Use estimated_document_count or count_documents instead. Please note that $where must be replaced by $expr, $near must be replaced by $geoWithin with $center, and $nearSphere must be replaced by $geoWithin with $centerSphere
  count = users.count()
2021-10-19 22:06:22,981 - telegram.ext.dispatcher - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/telegram/ext/dispatcher.py", line 432, in process_update
    handler.handle_update(update, self, check, context)
  File "/usr/local/lib/python3.7/site-packages/telegram/ext/conversationhandler.py", line 517, in handle_update
    new_state = handler.handle_update(update, dispatcher, check_result, context)
  File "/usr/local/lib/python3.7/site-packages/telegram/ext/handler.py", line 156, in handle_update
    return self.callback(update, context)
  File "bot.py", line 271, in follow_twitter
    update.message.reply_text(text=FOLLOW_TWITTER_TEXT, parse_mode=telegram.ParseMode.MARKDOWN)
  File "/usr/local/lib/python3.7/site-packages/telegram/message.py", line 603, in reply_text
    return self.bot.send_message(self.chat_id, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/usr/local/lib/python3.7/site-packages/telegram/bot.py", line 135, in decorator
    result = func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/telegram/bot.py", line 466, in send_message
    api_kwargs=api_kwargs,
  File "/usr/local/lib/python3.7/site-packages/telegram/bot.py", line 281, in _message
    result = self._post(endpoint, data, timeout=timeout, api_kwargs=api_kwargs)
  File "/usr/local/lib/python3.7/site-packages/telegram/bot.py", line 245, in _post
    return self.request.post(f'{self.base_url}/{endpoint}', data=data, timeout=timeout)
  File "/usr/local/lib/python3.7/site-packages/telegram/utils/request.py", line 354, in post
    **urlopen_kwargs,
  File "/usr/local/lib/python3.7/site-packages/telegram/utils/request.py", line 272, in _request_wrapper
    raise BadRequest(message)
telegram.error.BadRequest: Can't parse entities: can't find end of the entity starting at byte offset 57
```

**Comment by trizin** [comment]
Have you modified the python code? Could you share the first 5 lines of your .env file so I can try to reproduce the issue?
https://github.com/trizin/Telegram-Airdrop-Bot/blob/e4296bd90b39e17c7225a684834d651d3d20a476/.env.example#L1-L5

**Comment by bigfootdeveloper** [comment]
I found the error.  Some characters in the links broke the code.  

In my case, it was a low dash in the twitter link.  

I escaped it and works well.  I think that it will be good to put in the documentation or parse the links automatically in the functions. 

TWITTER_LINKS=https://twitter.com/bigfoot\_finance
TELEGRAM_LINKS=https://t.me/BigfootFinances

Thanks a lot for your good work and for release the code.
